utils/utils.go

```diff
@@ -29,11 +29,12 @@ func ExecCmd(name string, args ...string) (string, error) {
 }
 
 // ExecCmdWithStdStreams execute a command with the specified standard streams.
-func ExecCmdWithStdStreams(stdin io.Reader, stdout, stderr io.Writer, name string, args ...string) error {
+func ExecCmdWithStdStreams(env []string, stdin io.Reader, stdout, stderr io.Writer, name string, args ...string) error {
```

Comment:
@baude - Didn't you make this change in another of your PRs? Might want to hash this out to try and avoid breaking each other